OctopusProjectBuilder.Console/Program.cs

```diff
@@ -55,10 +66,167 @@ private static async Task UploadDefinitions(Options options)
 
         private static async Task DownloadDefinitions(Options options)
         {
-            var model = await new ModelDownloader(await BuildRepository(options), _loggerFactory).DownloadModel();
-            new YamlSystemModelRepository(_loggerFactory).Save(model, options.DefinitionsDir);
+            var repository = await BuildRepository(options);
+            var model = await new ModelDownloader(repository, _loggerFactory)
+                .DownloadModel(options.ProjectName);
+
+            await new YamlSystemModelRepository(_loggerFactory).Save(model, options.DefinitionsDir, async yaml =>
+            {

```

Comment:
Please extract the Normalisation logic out of the Program class to make `DownloadDefinitions()` easy to follow.